run-system-tests/action.yml

```diff
@@ -141,14 +135,22 @@ runs:
         echo "INSTALL REQUIRED DEPENDENCIES"
         echo "======================================================"
         echo
-        echo "installing youtube-dl..."
+        curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/yarn.gpg
```

Comment:
Please echo whatever this is trying to do

Reply:
fixed